t/03-testapi.t

```diff
@@ -682,6 +690,7 @@ subtest 'compat_args' => sub {
 subtest 'check quiet option on script runs' => sub {
     $bmwqemu::vars{_QUIET_SCRIPT_CALLS} = 1;
     my $mock_testapi = Test::MockModule->new('testapi');
+    $mock_testapi->redefine(script_output => sub { return 'output'; });
```

Comment:
```suggestion
    $mock_testapi->redefine(script_output => 'output');
```

Reply:
done

Reply:
I still see the old code...

Reply:
Me too. I think you only changed https://github.com/os-autoinst/os-autoinst/pull/1974/files#diff-97e29b53bd6b0cc520266e30da42f72439278f368a9d72db83678f535f1c8c63R567

Reply:
ups, yeah. I didn't touched this part, thats why I did not changed it. 
Np I will clean this part up too.